pandas/_libs/tslibs/offsets.pyx

```diff
@@ -418,6 +510,31 @@ cdef inline int month_add_months(pandas_datetimestruct dts, int months) nogil:
     return 12 if new_month == 0 else new_month
 
 
+def shift_day(datetime other, int days):
+    """
```

Comment:
def?

Reply:
At the moment it’s only called from python-land.  When that changes we’ll cpdef it.